src/posts/lucky_0.23.cr

```diff
@@ -0,0 +1,219 @@
+    will be stripped automatically. If you need to get the untouched value, you can use `params.get_raw`.
+
+    ### Memoize 2.0
+
+    The `Memoizable` module got a nice little upgrade. It now supports arguments, as well as
+    `false` and `nil` values. 
+
+    ```crystal
+    memoize user(user_id : Int64) : User?
+      UserQuery.new.find(user_id)
+    end
+    ```
+
+    Now when you call this method, it will run the first time, and return the value even if that value is
+    `nil`. If you need access to the uncached version you can use the `__uncached` version of the method.
+
+    ```crystal
+    # run the query once, and always return that value
+    user(4)
+
+    # always run the query and return the value
+    user__uncached(4)
+    ```
+
+    ### Security and more
+    
+    Some of the HTML page helpers like `highlight`, and `simple_format` would allow for unescaped HTML.
+    This could potentially be a security issue, so that's been fixed. All of these methods will escape the
+    HTML by default now. If you need the unescaped version, you can pass `escape: false`.
+
+    ```crystal
+    highlight("<p>This is a beautiful morning, but also a beautiful day</p>", "beautiful", escape: false)
+    ```
+
+    We've also fixed a few issues related to cookies.
+    * You can now set a `samesite` default.
+    * Exceptions aren't raised when trying to delete a cookie that doesn't exist
+    * Cookies are written at the correct time
+
+    ## Routing Updates
+
+    We have added a new `route_prefix` macro that alllows you to prefix your routes with
+    some path. This really helps with APIs where you may want to prefix all your routes with
+    something like `/api/v1`.
+
+    ```crystal
+    abstract class ApiAction < Lucky::Action
+      accepted_formats [:json], default: :json
+
+      route_prefix "/api/v1"
+    end
+
+    class Posts::Index < ApiAction
+      # GET /api/v1/posts`
+      get "/posts" do
+        #...
+      end
+    end
+    ```
+
+    Sometimes you may have a route with optional path params. For example, imagine a list of
+    blog posts that can be optionally filtered by appending the year, or year and month.
+
+    ```crystal
+    class Posts::Index < BrowserAction
+      # Matches GET /posts
+      # GET /posts/2020
+      # GET /posts/2020/07
+      get "/posts/?:year/?:month" do
+        # ...
+      end
+    end
+    ```
+
+    On top of all the neat routing features, we were also able to squeeze a little more performance
+    out thanks to [@matthewmcgarvey](https://github.com/luckyframework/lucky_router/pull/26)!
+
+    ## CLI Task Args
+
+    Writing CLI tasks for your application are pretty common, but they can also be a bit complicated.
+    Lucky already comes with a ton of tasks like `lucky db.migrate`, and `lucky gen.page About`. When
+    you start to write your own, you may need to pass in arguments from the CLI. We break these in to 3 different type.
```

Comment:
What do you think about making the association between these sentences a bit clearer somehow. Maybe something like this? 

> Lucky has many built-in tasks like `lucky db.migrate` and `lucky gen.page`, but often times an application needs their own custom tasks. Now Lucky makes it easier than ever to write these types of tasks

That doesn't quite work, but maybe sparks some ideas? Or maybe we cut it out and skip straight to something like: "Lucky now makes it easier than ever to write custom CLI tasks with arguments" and assume people will know what that means?